browser-profiler/dom-bridger/index.ts

```diff
@@ -72,7 +96,8 @@ function isDevtoolsIndicator(path: string, propertyName: string): boolean {
   if (devtoolsIndicators.added.some(pattern => pathIsPatternMatch(path, pattern))) return true;
   if (devtoolsIndicators.extraAdded.some(pattern => pathIsPatternMatch(path, pattern))) return true;
   if (devtoolsIndicators.changed.some(pattern => pathIsPatternMatch(path, pattern))) return true;
-  if (devtoolsIndicators.extraChanged.some(pattern => pathIsPatternMatch(path, pattern))) return true;
+  if (devtoolsIndicators.extraChanged.some(pattern => pathIsPatternMatch(path, pattern)))
```

Comment:
eslint is putting this one the nextline, but multiline if without {} seems weird, not sure what you think about that?

Reply:
It's probably hitting some kind of max line limit. I don't like it, but seems unnecessary to fix eslint too as part of this